src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Fixtures/PublicationRedirectGeneratorExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+
+public static class PublicationRedirectGeneratorExtensions
+{
+    public static Generator<PublicationRedirect> DefaultPublicationRedirect(this DataFixture fixture)
+        => fixture.Generator<PublicationRedirect>().WithDefaults();
+
+    public static Generator<PublicationRedirect> WithDefaults(this Generator<PublicationRedirect> generator)
+        => generator.ForInstance(d => d.SetDefaults());
+
+    public static InstanceSetters<PublicationRedirect> SetDefaults(this InstanceSetters<PublicationRedirect> setters)
+        => setters
+            .SetDefault(p => p.Slug)
+            .SetDefault(p => p.Created);
+
+    public static Generator<PublicationRedirect> WithPublication(
+        this Generator<PublicationRedirect> generator,
+        Publication publication)
+        => generator.ForInstance(s => s.SetPublication(publication));
+
+    public static Generator<PublicationRedirect> WithPublicationId(
+        this Generator<PublicationRedirect> generator,
+        Guid publicationId)
+        => generator.ForInstance(s => s.SetPublicationId(publicationId));
+
+    public static InstanceSetters<PublicationRedirect> SetPublication(
+        this InstanceSetters<PublicationRedirect> setters,
+        Publication publication)
+        => setters.Set(pr => pr.Publication, publication)
+            .SetPublicationId(publication.Id);
+
+    public static InstanceSetters<PublicationRedirect> SetPublicationId(
+        this InstanceSetters<PublicationRedirect> setters,
+        Guid publicationId)
+        => setters.Set(rr => rr.PublicationId, publicationId);
```

Comment:
You could use `pr` for `PublicationRedirect` or `p` for `Property` here. I think `rr` is an abbreviation of `ReleaseRedirect` probably copied over from `RedirectGeneratorExtensions`.

Reply:
Yep, good catch! I've changed this now :)